### ISSUE TITLE
File Browser Font Size

### ISSUE THREAD

**Issue by Mark-Agent003** [LMMS/lmms#4130]
This was originally included in #4085, but was removed so that that PR could focus on the highlights.
My proposal is that the `font-size` in the file browser be changed to 14px. This is a simple change that goes a long way. The file browser right now uses a 12px font size, and looks too small and tightly packed. Increasing the font size looks cleaner and adds a lot towards visibility.
Here are before and after pictures:
![image](https://user-images.githubusercontent.com/25230510/35404715-923b80e8-01c9-11e8-950b-2c31cff74592.png)
I'd encourage you to edit your style.css file to include ```QTreeView { font-size: 14px; }``` and try it out for a couple days before commenting.

**Comment by Umcaruje** [comment]
l would suggest using a relative unit like em because of hidpi support

On 25 Jan 2018 19:21, "Mark-Agent003" <notifications@github.com> wrote:

This was originally included in #4085
<https://github.com/LMMS/lmms/pull/4085>, but was removed so that that PR
could focus on the highlights.
My proposal is that the font-size in the file browser be changed to 14px.
This is a simple change that goes a long way. The file browser right now
uses a 12px font size, and looks too small and tightly packed. Increasing
the font size looks cleaner and adds a lot towards visibility.
Here are before and after pictures:
[image: image]
<https://user-images.githubusercontent.com/25230510/35404715-923b80e8-01c9-11e8-950b-2c31cff74592.png>
I'd encourage you to edit your style.css file to include QTreeView {
font-size: 14px; } and try it out for a couple days before commenting.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/LMMS/lmms/issues/4130>, or mute the thread

**Comment by Mark-Agent003** [comment]
@Umcaruje, font size only supports px and pt (at least for Qt)

**Comment by tresf** [comment]
Yeah, I don't see a good solution to this from Qt.  Here's an article sort-of explaining it... <http://doc.qt.io/qt-5/scalability.html#calculating-scaling-ratio>

**Comment by musikBear** [comment]
One thing i like to add. 
If the wideness of the list-bar, is too narrow for the *whole* preset-name to be shown, a strange drop in font-size happens:
![fontsizetowidth](https://user-images.githubusercontent.com/6662066/35453149-6bf94c3e-02c9-11e8-80b5-bba879195d5e.JPG)

sufficient width : green
insufficient width : red
